lib/integrations/drivers/github.rb

```diff
@@ -0,0 +1,58 @@
+### Github integration
+module Integrations::Drivers
+  class Github < Integrations::BaseDriver
+    def self.display_name
+      'Github'
+    end
+
+    def type
+      :github
+    end
+
+    def api_url
+      'https://api.github.com/'
+    end
+
+    def create_task(title)
+      configuration = self.configuration
```

Comment:
Do we really need to use `self.` here? I think we can use `configuration` directly and not even make a variable.